hvantk/tests/test_expression_atlas_downloader.py

```
@@ -0,0 +1,80 @@
+import os
+from pathlib import Path
+
+import pytest
+from click.testing import CliRunner
+from hvantk.commands import expression_atlas_downloader
+from hvantk.tests.test_create_gene_annotation_tables import TEST_DIR
+
+TEST_DIR = Path(__file__).parent.parent
```

Ruff (0.8.2)

Redefinition of unused `TEST_DIR` from line 7

(F811)